config.rb

```diff
@@ -19,7 +19,7 @@
 set :markdown, fenced_code_blocks: true, smartypants: true
 set :markdown_engine, :redcarpet
 set :relative_links, true
-set :site_name, 'Adarsh Pandit Blog'
+set :site_name, 'Adarsh Pandit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.